src/constructorio.js

```diff
@@ -47,8 +48,7 @@ class ConstructorIO {
       sessionId,
       userId,
       fetch,
-      trackingSendDelay,
-      sendTrackingEvents,
+      requestQueue,
```

Comment:
If we are aiming for speed, let's keep the existing interface as changing the name will result in changes in other repos also.  If we are aiming for correctness, maybe rename to `requestQueue` to `trackingOptions`?

Reply:
> If we are aiming for speed, let's keep the existing interface as changing the name will result in changes in other repos also. If we are aiming for correctness, maybe rename to `requestQueue` to `trackingOptions`?

I figure we need to make a change anyway to explicitly turn on `sendTrackingEvents`, so it would be a good time to do this refactor now.

>  If we are aiming for correctness, maybe rename to requestQueue to trackingOptions

The event emitter options are currently named after the class - was following the same pattern. trackingOptions works for me, though it's a bit odd that idOptions is separate from this.